docs/colors/components/palette.constants.ts

```diff
@@ -0,0 +1,164 @@
+const PURPOSE = {
+  title: 'Purpose',
+  content: [
+    'Colors communicate branding. When applied consistently , they can communicate information about system states, and draw attention to certain parts of the screen, like an error that needs attention.',
+    'Though color does much to make a design more efficient and effective, we must still pursue designs that are effective even to those with little or no color perception. Therefore, colors should be used in an accessible manner, and no information should be communicated by color alone.',
+  ],
+};
+
+const COLOR_CONTRAST_EXPLANATION = [
+  'All colors have been analyzed against each other in this matrix, and each combination given a score. Every color in our palette has AA',
+  'AA or AAA: The two colors can be used as foreground and background against each other with no restrictions. AA18: Can be used for graphical elements, icons, or for text, as long as the text size is at least 18 pt. (Or 14 if bold) DNP: Do not use this combination for anything functional or essential. An exception might be a decorative object like a divider, or for a use in which the information is communicated in an additional way..',
+  'Some examples of color combinations that are inaccessible.',
+];
+
+const COLORS = [
+  {
+    title: 'Core Neutrals',
+    content: 'The “paper” colors are our near-black and near-black base backgrounds.',
+    colors: [
+      [
+        { token: 'Common-Black', bgColor: '#000', color: '#FFF' },
+        { token: 'Common-White', bgColor: '#FFF', color: '#000' },
+      ],
+    ],
+  },
+  {
+    content:
+      'When the user switches between light and dark themes, most of the change will show itself via a dark grey changing to a light, or a light changing to a dark.',
+    colors: [
+      [
+        { token: 'Paper-Black', bgColor: '#323232', color: '#FFFFFF' },
+        { token: 'Paper-White', bgColor: '#F9F9F9', color: '#323232' },
+      ],
+    ],
+  },
+  {
+    content:
+      'Often, but not always, a grey will change to a complimentary one on the opposite side of this spectrum.',
+    colors: [
+      [
+        { token: 'Grey-50', bgColor: '#F9F9F9', color: '#323232' },

```

Comment:
I think these should directly reference the `theme` palette :) .   Then we update there, it updates here.